LosGosusTest/src/Validator/BookValidatorTest.cs

```diff
@@ -0,0 +1,110 @@
+using LosGosus.Models;
+using LosGosus.Validators.Concretes;
+
+public class BookValidatorTests
```

Comment:
There is already a BookValidatorTests class on /services/BookValidatorTests.cs

Reply:
Guess he should delete it. That class was with a previous version of the refactor.